src/vm/arm/asmhelpers.S

```diff
@@ -1423,7 +1423,7 @@ DelayLoad_MethodCall:
     // Load the helper arguments
     ldr         r5, [sp,#(__PWTB_TransitionBlock+10*4)] // pModule
     ldr         r6, [sp,#(__PWTB_TransitionBlock+11*4)] // sectionIndex
-    ldr         r7, [sp,#(__PWTB_TransitionBlock+12*4)] // indirection
+    ldr         r8, [sp,#(__PWTB_TransitionBlock+12*4)] // indirection
```

Comment:
Do we have the same problem here as well?

https://github.com/dotnet/coreclr/pull/17275/files#diff-9370feb9d6964a4954cca434fea5c4bdL1470

Reply:
Yes, we do. Thank you for noticing that.

Reply:
Fixed that one too